app/lib/frontend/handlers/routes.dart

```diff
@@ -245,6 +245,11 @@ class PubSiteService {
   Future<Response> apiDocumentation(Request request, String package) =>
       apiDocumentationHandler(request, package);
 
+  @Route.post('/api/packages/<package>/flag/<flag>/<status>')
```

Comment:
I would suggest we do something like:

```
GET /api/packages/<package>/options
{
  'isDiscontinued': true | false,
}

PUT /api/packages/<package>/options
{
  'isDiscontinued': true | false
}
```

Then we strictly limit accepted input to `isDiscontinued` which must be bool. So that input with any other properties will be rejected. Then we can add more properties in the future.

If we make these nice, we can later declare them stable and perhaps augment the CLI client, etc.

Reply:
Why `options`? Although, I think I like it a bit more than `flags` as it would allow non-boolean values later.

Reply:
Yeah, allowing for other things in the future was the thinking... I'm not sure what those would be...

But things that you can change about a package without publishing a new version... and things that aren't package-version specific could be called _options_ :)